component/service/src/main/java/org/exoplatform/social/service/malwareDetection/connector/FileMalwareDetectionServiceConnector.java

```diff
@@ -0,0 +1,68 @@
+package org.exoplatform.social.service.malwareDetection.connector;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.exoplatform.commons.file.model.FileItem;
+import org.exoplatform.commons.file.services.FileService;
+import org.exoplatform.container.xml.InitParams;
+import org.exoplatform.services.log.ExoLogger;
+import org.exoplatform.services.log.Log;
+
+public class FileMalwareDetectionServiceConnector extends MalwareDetectionServiceConnector {
+  
+  private static final Log LOGGER = ExoLogger.getExoLogger(FileMalwareDetectionServiceConnector.class);
+  private FileService fileService;
+  
+  public FileMalwareDetectionServiceConnector(InitParams initParams, FileService fileService) {
+    super(initParams);
+    this.fileService = fileService;
+  }
+  
+  @Override
+  public List<Map<String, String>> getInfectedItems(String infectedFilePath) {
+    String fileChecksum = getInfectedFileChecksum(infectedFilePath);
+    List<Map<String, String>> infectedItems = new ArrayList<Map<String, String>>();
+    try {
+      List<FileItem> infectedFileItems = fileService.getFilesByChecksum(fileChecksum);
+      for (FileItem infectedFileItem : infectedFileItems) {
+        if (infectedFileItem != null && infectedFileItem.getFileInfo() != null) {
+          Map<String, String> infectedItem = new HashMap<String, String>();
+          infectedItem.put(INFECTED_ITEM_NAME, infectedFileItem.getFileInfo().getName());
+          infectedItem.put(INFECTED_ITEM_LAST_MODIFIER, infectedFileItem.getFileInfo().getUpdater());
+          infectedItems.add(infectedItem);
+        }
+      }
+    } catch (Exception e) {
+      LOGGER.error("Error when trying to get the infected item informations", e);
```

Comment:
shouldn't we log some identifier of the infected item to help troubleshoot ?

Reply:
yes I will rework the logs with the good format, it is just the draft :)

Reply:
I have added one log [here](https://github.com/Meeds-io/social/pull/618/files#diff-8c188b1765721990c9d74eed2edd9399e5a54bc0e38a7488a31da8df96fcf6beR38) when detecting a file as a malware